src/main/webapp/Portal.html

```diff
@@ -167,6 +167,10 @@
       loadCss(cdnEndpoint + 'generated/swc.css')
       loadJs(cdnEndpoint + 'generated/jquery.min.js')
       loadJs(cdnEndpoint + 'generated/jquery-ui.min.js')
+      // Load latest version of Bootstrap 3 to override the version bundled by GWTBootstrap3
+      // The Bootstrap 3 custom theme CSS is loaded through swc.scss
+      // Note: we cannot source this from NPM because we rely on Bootstrap 4 to be in node_modules/bootstrap to compile our scss.
+      loadJs(cdnEndpoint + 'js/bootstrap-3.4.1-dist/js/bootstrap.min.js')
```

Comment:
Is it possible to pull bootstrap 3.4.1 from npm in package.json and copy over the assets into the generated folder?

Reply:
I originally did not because because when we compile our CSS with Sass, it also uses scss source files from synapse-react-client (so scss variables can be overridden). synapse-react-client's scss relies on Bootstrap 4's scss to be present, so ultimately Bootstrap 4's scss must be located in `SynapseWebClient/node_modules/bootstrap`. 

We can work around this by using an [alias](https://classic.yarnpkg.com/lang/en/docs/cli/add/#toc-yarn-add-alias)